gnome-extra/cinnamon/files/cinnamon-6.2.0-polkit-agent-on-x11.patch

```diff
@@ -0,0 +1,75 @@
+From c6402be3b5eb84f53367ebd57b1a38664a240d44 Mon Sep 17 00:00:00 2001
+From: Sparky Bluefang <sparky@bluefang-logic.com>
+Date: Wed, 3 Jul 2024 02:10:18 -0400
+Subject: [PATCH] Use Polkit agent for X11 sessions. Don't crash if polkit
+ doesn't start properly.
```

Comment:
Worth noting that according to upstream, this doesn't work with multiple users.
https://github.com/linuxmint/cinnamon/pull/12272#issuecomment-2211964115

I'm not sure if that limitation out-weights cleaning-up the ebuild and removing a consumer of `polkit-gnome`.

It also looks like Fedora is shipping this change (without the error handling).

Reply:
My gut feeling is it's tolerable to have deprecated software used by one more package, until cinnamon gets their replacement fully implemented.